and-cli-workspace.js

```diff
@@ -159,27 +163,41 @@ require("./command-runner").run(async () => {
         )
         .parse(process.argv);
 
+    const cloneEnabled = program.clone != null;
+    const forkEnabled = program.fork != null;
+    const usernamesProvided = StringUtils.hasValue(program.usernames);
+
+    // If no flags provided, explain options
+    // -------------------------------------
+    if (!cloneEnabled && !forkEnabled && !usernamesProvided) {
```

Comment:
I believe we get this 'fall-through' behavior in other commands with something like this, unless we want something a little more in-your-face:

```JS
    if (process.argv.slice(2).length === 0) {
        program.help();
    }
```
(See `and-cli-migration.js` as an example)

We could still keep the variables used to run the individual functions broken out for readability

Reply:
Check it out again, I wrapped that in a function to make it more readable. Whatcha think?